Makefile

```diff
@@ -31,5 +31,11 @@ go-lint:
 generate:
 	go generate ./...
 
+.PHONY: check-alerts
+check-alerts:
+	# If you don't have promtool, install it with
+	# go install -a github.com/prometheus/prometheus/cmd/promtool@latest
```

Comment:
This is on purpose made as a comment to guide developers on how to obtain a tool, but do not install it outside of the project (it may replace tool used by the user).